.github/workflows/ci.yml

```diff
@@ -151,13 +155,13 @@ jobs:
         os: [windows-latest, macos-latest, ubuntu-latest]
         include:
           - os: ubuntu-latest
-            gpu: 'yes'
+            gpu: "yes"
```

Comment:
Why have you made this change?

Reply:
erg, just noticed this automatically came in again.. 

The top of the file uses double quotes, and my IDE formats on save, so it replaced these with double quotes for consistency.